pep-0514.txt

```diff
@@ -187,6 +226,7 @@ may look like this::
         SupportUrl = "http://www.example.com/distro-3"
         Version = "3.0.12345.0"
         SysVersion = "3.6.0"
+        SysArchitecture = "64bit"
 
```

Comment:
So a virtualenv user would use ```-p ExampleCorp\6C465E66``` to specify this distribution? That seems pretty user-hostile, and I'd expect them to just use ```-p <path to interpreter>``` instead, which pretty much defeats the purpose.

For an IDE, the above is probably OK. You present a "Select active Python" dialog, and display some of this data (Company DisplayName, tag DisplayName, Version, SysVersion and SysArchitecture would be about what I'd want to see) and the user selects the one they want.